build_ngtf.py

```diff
@@ -130,6 +131,12 @@ def main():
         "Local nGraph source directory to use. Overrides --ngraph_version.\n",
         action="store")
 
+    parser.add_argument(
+        '--tf_version',
+        type=str,
+        help="tensorflow version to use. (Default: " + tf_version + ")\n",
+        action="store")
```

Comment:
is this option used later on?
maybe in this line `cmd = cmdpart + ["-U", "tensorflow==" + tf_version]` you want `arguments.tf_version` instead of `tf_version`?

Reply:
if arguments.tf_version is assigned, I override tf_version with arguments.tf_version

Reply:
ah, right. missed that